dbt_subprojects/dex/models/_projects/uniswap/uniswap_tvl_daily.sql

```diff
@@ -0,0 +1,304 @@
+    , materialized = 'incremental'
+    , file_format = 'delta'
+    , incremental_strategy = 'merge'
+    , unique_key = ['blockchain', 'project', 'version', 'id', 'block_date']
+    , incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_date')]
+    , post_hook='{{ expose_spells(\'[
+                                      "ethereum","arbitrum","base","ink","blast","optimism","blast","bnb","zora","avalanche_c","unichain","worldchain"
+                                    ]\',
+                                    "sector",
+                                    "dex",
+                                    \'["Henrystats"]\') }}')
+}}
+
+{% if is_incremental() %}
+
+with
+
+daily_events as (
+    select 
+        block_month
+        , block_date
+        , blockchain
+        , project
+        , version
+        , id
+        , token0 
+        , token1 
+        , token0_symbol 
+        , token1_symbol 
+        , amount0_raw 
+        , amount1_raw 
+        , amount0 
+        , amount1
+    from 
+    {{ ref('uniswap_daily_agg_liquidity_events') }}
+    where {{ incremental_predicate('block_date') }}
+),
+
+min_daily as (
+    select 
+        min(block_date) as block_date 
+    from 
+    daily_events
+),
+
+tvl_min_daily as (
+    select 
+        block_month
+        , block_date
+        , blockchain
+        , project
+        , version
+        , id
+        , token0 
+        , token1 
+        , token0_symbol 
+        , token1_symbol 
+        , token0_balance_raw as amount0_raw 
+        , token1_balance_raw as amount1_raw 
+        , token0_balance as amount0 
+        , token1_balance as amount1
+    from 
+    {{this}}
+    where block_date = (select block_date from min_daily)
+),
+
+daily_events_final as (
+    select 
+        *
+        , 'include' as check_filter
+    from 
+    daily_events 
+    where block_date != (select block_date from min_daily)
+
+    union all 
+
+    select 
+        *
+        , 'exclude' as check_filter
+    from 
+    tvl_min_daily 
+),
+
+daily_cum as (
+    select 
+        * 
+        , sum(amount0_raw) over (partition by blockchain, project, version, id order by block_date asc) as token0_balance_raw
+        , sum(amount1_raw) over (partition by blockchain, project, version, id order by block_date asc) as token1_balance_raw
+        , sum(amount0) over (partition by blockchain, project, version, id order by block_date asc) as token0_balance
+        , sum(amount1) over (partition by blockchain, project, version, id order by block_date asc) as token1_balance
+        , lead(block_date, 1, current_timestamp) over (partition by blockchain, project, version, id order by block_date asc) as next_day
+    from 
+    daily_events_final 
+),
+
+time_seq as (
+    select
+        sequence(
+        cast('2025-01-01' as timestamp),
+        date_trunc('day', cast(now() as timestamp)),
+        interval '1' day
+        ) as time 
+),
```

Comment:
we have built a `utils.days` table in the daily subproject which is intended to replace the need for this syntax pattern. can you update here to read from that table? you can use normal jinja source syntax to refer to table, as it's established as a source in spellbook to be used in other dbt subprojects